packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.5.30
```

Comment:
Odd that you got changes from google_maps_flutter, what command did you use to merge/rebase your branch?

Reply:
First i merge changes to master branch using upstream and then made to merge it in my current branch , I used git rebase master and git merge origin/yash_device_info and then commited and 
pushed the changes

Reply:
Hmm, odd, normally you either rebase or merge after updating your origin/master (that's how I do it! :P)

We'll see when we merge this, maybe you become the forever owner of methods in other plugins ;)

Reply:
(Don't worry too much, I think github is going to be able to manage :P)

Reply:
Yes I did updated my origin/master first and then merged the changes in the local branch. I don't know what went wrong. Let me know if i can do anything to fix this.